packit/api.py

```diff
@@ -252,7 +252,35 @@ def pkg_tool(self) -> str:
 
         return self.config.pkg_tool
 
-    def sync_release_env(self, version: Optional[str] = None):
+    def common_action_env(self, version: Optional[str] = None):
```

Comment:
```suggestion
    def common_actions_env(self, version: Optional[str] = None):
```

but I am ok also with the current one